contentcuration/contentcuration/utils/publish.py

```diff
@@ -403,7 +403,15 @@ def process_assessment_metadata(ccnode, kolibrinode):
     randomize = exercise_data.get('randomize') if exercise_data.get('randomize') is not None else True
     assessment_item_ids = [a.assessment_id for a in assessment_items]
 
-    mastery_model = {'type': exercise_data.get('mastery_model') or exercises.M_OF_N}
+    exercise_data_type = ""
+    if exercise_data.get('mastery_model'):
+        exercise_data_type = exercise_data.get('mastery_model')
+    if exercises.M_OF_N:
```

Comment:
This appears like it would always be true? And thus, wouldn't replicate the existing behavior. Some unit tests could help

Reply:
`exercises.M_OF_N` is a constant imported from `le_utils` -- `from le_utils.constants import exercises`

Reply:
I change this and now it should replicate the same behavior as before for the new model